platform/ios/app/MBXViewController.m

```diff
@@ -2366,6 +2366,25 @@ - (void)mapView:(nonnull MGLMapView *)mapView didChangeLocationManagerAuthorizat
     }
 }
 
+- (MGLUserLocationStyle)mapViewStyleForUserLocationView:(MGLMapView *)mapView {
```

Comment:
How would you envision this working if the developer wanted to dynamically change the default user location annotation color? When we make an `ios-sdk-examples` entry for this, we should highlight this kind of use case.